borg/keymanager.py

```diff
@@ -103,16 +103,16 @@ def grouped(s):
         export += 'BORG PAPER KEY v1\n'
         lines = (len(binary)+17) // 18
         repoid = hexlify(self.repository.id).decode('ascii')[:18]
-        complete_checksum = sha256_hexdigest(binary)[:12]
+        complete_checksum = sha256_hexdigest(binary, 12)
```

Comment:
as it is not really a (full) sha256 any more, maybe just call it hexdigest or hash_hex - also shorter to type?